sorc/machine-setup.sh

```diff
@@ -109,6 +109,15 @@ elif [[ -d /lustre && -d /ncrc ]] ; then
     fi
     target=gaea
     module purge
+elif [[ "$(hostname)" =~ "Orion" ]]; then
```

Comment:
Do the quotations around the target variable (e.g., target="odin") matter?  I don't see them for the other machines, such as target=gaea, or target=hera.

Reply:
Most of the time, it will not matter unless the variable _$(hostname)_ returns a strange values (containing whitespaces or wildcards) in rarely cases instead of alphanumeric only. It is usually a good practice to quote a string values because it is safer.

Reply:
Thanks for the clarification!  We should leave it as is then.